pallets/gear/rpc/runtime-api/src/lib.rs

```diff
@@ -20,14 +20,17 @@
 #![doc(html_logo_url = "https://docs.gear.rs/logo.svg")]
 #![doc(html_favicon_url = "https://gear-tech.io/favicons/favicon.ico")]
 
-pub use pallet_gear::{manager::HandleKind, GasInfo};
+pub use pallet_gear::{manager::HandleKind, GasInfo, ReadOnlyReply};
 use sp_core::H256;
 use sp_runtime::traits::Block as BlockT;
 use sp_std::vec::Vec;
 
 sp_api::decl_runtime_apis! {
-    #[api_version(2)]
+    #[api_version(3)]
     pub trait GearApi {
+        #[allow(clippy::too_many_arguments)]
+        fn read_only_send_message(origin: H256, destination: H256, payload: Vec<u8>, gas_limit: u64, value: u128, allowance_multiplier: u64) -> Result<ReadOnlyReply, Vec<u8>>;
```

Comment:
to consider:  I think better name is `send_message_read_only`, because we have already `send_message` and this is the same action (but rpc) with some specific - `read_only`.

Reply:
Yeah, good suggestion, but actually I dont like this name at all, but couldn't imagine any better xd